zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/ElasticsearchExporterTest.java

```diff
@@ -376,6 +376,15 @@ void shouldForbidNegativeNumberOfReplicas() {
       // when - then
       assertThatCode(() -> exporter.configure(context)).isInstanceOf(ExporterException.class);
     }
+
+    @Test
+    void shouldForbidNegativePriority() {
+      // given
+      config.index.setTemplatePriority(-1);
+
+      // when - then
+      assertThatCode(() -> exporter.configure(context)).isInstanceOf(ExporterException.class);
```

Comment:
🔧 Might be worth strengthening the assertion to include the message as the method under test can throw this exception for multiple reasons